subconscious/model.py

```diff
@@ -24,23 +42,27 @@ def __init__(self, type=str, primary_key=None, composite_key=None, index=None,
         You can't have both a primary_key and composite_key in the same model.
         index is whether you want this column indexed or not for faster retrieval.
         """
-        # TODO: support for other field types (datetime, etc)
+        if type not in (str, int):
+            # TODO: support for other field types (datetime, uuid, etc)
+            err_msg = 'Bad Field Type: {}'.format(type)
+            raise InvalidColumnDefinition(err_msg)
+
+        if primary_key and composite_key:
```

Comment:
It seems weird to me that there are both `primary_key` and `composite_key` parameters here. I know that's not really a nit in the scope of this change and it might be the pythonic way, but thought I'd mention it.